bgpd/bgp_vty.c

```diff
@@ -14606,28 +14606,6 @@ CPP_NOTICE("bgpd: remove deprecated 'ip large-community-list <(1-99)|(100-500)|s
 #endif
 DEFUN (lcommunity_list_standard,
        bgp_lcommunity_list_standard_cmd,
-       "bgp large-community-list (1-99) <deny|permit>",
```

Comment:
```
#if CONFDATE > 20191005
CPP_NOTICE("bgpd: remove deprecated 'ip large-community-list <(1-99)|(100-500)|standard|expanded> <deny|permit> <LINE|AA:BB:CC>' command")
#endif
```
This is here because we have a 1 year deprecation period for CLI. The period expires October 05.

 Since our next release is due sometime in October, I am fine with removing the CLI right now. However, these warnings need to be removed too, they're tied to the CLI.

Reply:
Thanks, Quentin(@qlyoung).
These changes do not remove the deprecated `ip large-community-list ...`.
Instead, the changes only removes the CLI definitions that allow erroneous configurations.
So, we should still retain the warnings untill all the deprecated CLI definitions are removed completely.